app/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -350,7 +350,7 @@ def test_update_experiment_branches_with_feature_config(self):
         self.assertEqual(experiment.feature_configs.get(), feature)
         self.assertEqual(experiment.branches.count(), 2)
         self.assertEqual(experiment.reference_branch.name, reference_branch_data["name"])
-        self.assertEqual(experiment.reference_branch.feature_values.get().enabled, False)
+        self.assertEqual(experiment.reference_branch.feature_values.get().enabled, True)
```

Comment:
Tiny little nit, feel free to ignore: should we add "mobile" to the test name here? i.e. is there any reason to test for feature configs in both the desktop and mobile cases?

Reply:
Ahh, I see you added a mobile test in the other file. Can we add "desktop" to this one then?

Reply:
Oh yeah, this one is mobile, I'll add it to the name 👍

Reply:
And yeah now the feature configs behave differently between desktop and mobile so we need separate tests.